plugin/action.yml

```diff
@@ -56,9 +61,25 @@ runs:
         pipx install "poetry$POETRY_VERSION"
         pipx inject poetry "coveo-stew$COVEO_STEW_VERSION"
 
-    - name: Run stew ci
-      if: inputs.run-stew == 'true'
+    - name: Target all projects in the repository
+      if: inputs.run-stew == 'true' && inputs.project-name == ''
+      shell: bash
+      run: echo "Stew will run on all projects in the repository."
```

Comment:
Should we do something like `echo "STEW_CI_ARGS=''" >> $GITHUB_ENV` to be sure that we dont get the value `STEW_CI_ARGS` from somewhere else? 🤔